src/scheduler/workers/unpublish/weekly/sendUnpublishReminder.ts

```diff
@@ -0,0 +1,70 @@
+import { ListItemJsonData } from "server/models/listItem/providers/deserialisers/types";
+import { schedulerLogger } from "scheduler/logger";
+import { NotifyClient, RequestError } from "notifications-node-client";
+import { NOTIFY } from "server/config";
+import { Meta } from "./types";
+import { addUnpublishReminderEvent } from "./addUnpublishReminderEvent";
+import { weeklyReminderPersonalisation } from "./weeklyReminderPersonalisation";
+import { ListItem } from "@prisma/client";
+
+const template = NOTIFY.templates.annualReviewNotices.providerStart;
+
+const logger = schedulerLogger.child({ method: "sendUnpublishEmail", template });
+const notifyClient = new NotifyClient(NOTIFY.apiKey);
+
+export async function sendUnpublishReminder(listItem: ListItem, meta: Meta) {
+  const jsonData = listItem.jsonData as ListItemJsonData;
+  const personalisation = weeklyReminderPersonalisation(listItem, meta);
+  const emailAddress = jsonData.emailAddress;
+
+  logger.silly(`${JSON.stringify(personalisation)}, email address ${emailAddress}`);
```

Comment:
well, I would use `trace` because it's a more descriptive method but it doesn't look like Winston supports it. 

There are a lot of debug/info logs that look like this.. I first had this as logger.debug but it was far too noisy. I've tried to follow similar form, but I don't really need them. it's a bit like doing `--verbose` when you _really_ want to know what's going on step by step. 

I believe our environments are set to info or higher. You wouldn't ordinarily see these logs unless you set the LOG_LEVEL.